andela/wg-wits
# #159236203 Relate WorkoutSession & WorkoutLog Models.

#### What does this PR do?
- This PR creates a foreign key `session_id` on the WorkoutLog model referencing the WorkoutSession 
### How should this be manually tested?
-  Clone the application as guided in the README.
- Create virtual environment and activate it.
- `pip install -r requirements_devel.txt`
- `pip install psycopg2_binary`
- `-invoke create-settings --settings-path ./settings.py`
- `npm install bower`
- `sudo python manage.py bower_install --allow-root`
- `invoke bootstrap-wger --settings-path ./settings.py --no-start-server`
- `git checkout ch-wrkoutlog-sessn-rlation-159236203`
- Make and run the migrations by:
         - `python manage.py makemigrations`
         - `pyothn manage.py migrate`
- Login the application as any user and add a workout  at `http://127.0.0.1:8000/en/workout/overview`
- Add a weight log to that day and check out the database tables `manager_workoutlog` and `manager_workoutsession`. Notice the primary key on the `manager_workoutsession` table being referenced by the `session_id` column on the `manager_workoutlog` table.
#### Relevant Pivotal Tracker story.
[#159236203](https://www.pivotaltracker.com/story/show/159236203)
